tutorials/auto_scheduler/tune_matmul_x86.py

```diff
@@ -147,22 +156,10 @@ def matmul_add(N, L, M, dtype):
 # file "matmul.json". The measurement records can be used to re-apply search results,
 # resume the search, and perform other analyses.
 
-######################################################################
-# Here is an example where we load the best schedule from a file,
-# print the equivalent python schedule API, and build the binary again.
-
-# Load the measuremnt record for the best schedule
-inp, res = auto_scheduler.load_best(log_file, task.workload_key)
```

Comment:
Better to remind readers that we use `task.apply_best` to load and apply the best schedule, or it looks a bit weird to only see how to print the best schedule here.